source/guides/deployment.rst

```diff
@@ -0,0 +1,162 @@
+:orphan:
+
+Deploy Mattermost
+===================
+
+Learn how to install, deploy, and scale Mattermost for teams and organizations of any size.
+
+
+Get Started
+-----------
+
+These guides will get you up and running with Mattermost in minutes.
+
+
+
+* :doc:`Run Mattermost via Docker </install/setting-up-local-machine-using-docker>` - Launch a Mattermost server instantly to test functionality and build integrations.
+* :doc:`Quick Install Guide </getting-started/light-install>` - Deploy in minutes via Mattermost Omnibus on Ubuntu.
+* :doc:`Administrator Tasks </getting-started/admin-onboarding-tasks>` - Learn about the standard configurations and settings you’ll encounter.
+* :doc:`Architecture </getting-started/architecture-overview>` - Learn the basics of user authentication, notifications, data management services, network connectivity, and high availability.
+* :doc:`Implement Mattermost </getting-started/implementation-plan>` - Get a detailed breakdown of the technical requirements to deploy Mattermost for your team or organization.
+* :doc:`Enterprise Roll Out Checklist </getting-started/enterprise-roll-out-checklist>` - Learn how to roll Mattermost out to thousands of users.
+* :doc:`Welcome Email Template </getting-started/welcome-email-to-end-users>` - Use our sample email template when you’re ready to invite users to your server.
+
+
+Install Guides
+--------------
+
+The Mattermost server and client apps can run on all of the most popular platforms. Here’s a list of the most popular installation methods.
+
+**Server Installation**
+
+
+
+* :doc:`Software and Hardware Requirements </install/software-hardware-requirements>`
+* :doc:`Local Docker Setup </install/setting-up-local-machine-using-docker>`
+* :doc:`Mattermost Omnibus </install/installing-mattermost-omnibus>`
+* :doc:`Ubuntu 20.04 LTS </install/installing-ubuntu-2004-LTS>`
+* :doc:`Kubernetes </install/install-kubernetes>`
+* `Helm <https://docs.mattermost.com/install/install-kubernetes.html#installing-the-operators-via-helm>`__
```

Comment:
I couldn't figure out how to create this anchor link without doing an external link. I'm open to alternatives if someone knows how to properly do this.

Reply:
This feels like a reasonable approach. Any concerns from your end, @justinegeffen?

Reply:
No concerns on my end - I know we can use a :ref: [as mentioned here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role) and we might want to look into that going forward but I don't think it's a blocker. :)

Reply:
I tried every combination of :ref: I could think of and nothing seemed to work properly. I have a suspicion that it could be related to how the target doc page is generated (it's 4 docs pages in one), but I wasn't able to diagnose the problem.

Reply:
Ah ok that makes sense. Thanks Ben! :)